app.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const app = express();
+const { getCategories } = require("./controllers/categoriesController");
+
+app.use(express.json());
+
+app.get("/api", (req, res) => {
+  res.status(200).send({ message: "Server online." });
+});
+
+app.get("/api/categories", getCategories);
+
+const server = app.listen("9090", () => {
```

Comment:
This is what you can extract - and subsequently you won't need to hold the return value